test/pstext/latext.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
```

Comment:
Note that those tests will fail in our CI, because we don't have latex installed.

Reply:
Can we flag that script to not run in the CI perhaps?

Reply:
I think we can check if `latex` and `dvips` are available. If not (for CI), then exclude these scripts from tests.